samza-core/src/main/java/org/apache/samza/system/RemoteExecutionEnvironment.java

```diff
@@ -24,7 +24,11 @@
 /**
  * This class implements the {@link ExecutionEnvironment} that runs the applications in YARN environment
```

Comment:
cc @xinyuiscool since this should probably be done while renaming to ApplicationRunner